src/entrypoint.sh

```diff
@@ -39,6 +39,9 @@ git config --global user.email "github-action@actions-template-sync.noreply.${SO
 git config --global user.name "${GITHUB_ACTOR}"
 git config --global pull.rebase false
 git config --global --add safe.directory /github/workspace
+
+git config --global "credential.https://${SOURCE_REPO_HOSTNAME}.helper" "!gh auth git-credential"
```

Comment:
@msiebeneicher thanks a lot for the PR.
Reading I think it does not break the `ssh`  functionality. Did you by any chance test the script with `ssh`?

Reply:
**Remark**
Created an issue for automated testing #241

Reply:
> Reading I think it does not break the ssh functionality. Did you by any chance test the script with ssh?

Unfortunately I have no possibilities to check ssh connections or functionalities.

> Is it possible that the code part throws if the token does not have sufficient permissions for the source repository?

I need to test this. It should be possible to check the `x-oauth-scopes` via own api call: https://github.com/orgs/community/discussions/25259

Reply:
>> Is it possible that the code part throws if the token does not have sufficient permissions for the source repository?
>> 
> I need to test this. It should be possible to check the x-oauth-scopes via own api call: https://github.com/orgs/community/discussions/25259 

Because of the two tokens now, it's pretty hard to have a separate check for the right token permissions. I would handle this as separate feature later in case of this is still useful. Two additional api requests to "fail fast" feels a bit overengineered for me at this point.